<?xml version="1.0"?>
<launch>
  
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" respawn="false" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="camera" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="test_signs"  pkg="wr8_ai"  type="test_signs.py" output="screen" respawn="false" >
    <param name="graph_path" value="$(find wr8_ai)/neural_networks/TSD/keras-yolo3/best/MobileNetv2_35.ncsg"/>  
    <param name="config_path" value="$(find wr8_ai)/neural_networks/TSD/keras-yolo3/best/rf_signs_mobilev2.json"/>  

    <remap from="camera" to="usb_cam/image_raw" />
  </node>

</launch>
